src/main/java/uk/gov/hmcts/ccd/domain/service/search/elasticsearch/security/ElasticsearchSecurityClassificationFilter.java

```diff
@@ -20,17 +21,23 @@
 
     private final UserRepository userRepository;
     private final CaseTypeService caseTypeService;
+    private final ApplicationParams applicationParams;
 
     @Autowired
     public ElasticsearchSecurityClassificationFilter(@Qualifier(CachedUserRepository.QUALIFIER)
                                                              UserRepository userRepository,
-                                                     CaseTypeService caseTypeService) {
+                                                     CaseTypeService caseTypeService,
+                                                     ApplicationParams applicationParams) {
         this.userRepository = userRepository;
         this.caseTypeService = caseTypeService;
+        this.applicationParams = applicationParams;
     }
 
     @Override
     public Optional<QueryBuilder> getFilter(String caseTypeId) {
+        if (applicationParams.getEnableAttributeBasedAccessControl()) {
+            return Optional.empty();
+        }
```

Comment:
Might be nicer instead to use `@ConditionalOnProperty` on the class level so the bean doesn't even get created when AC is enabled.